var/spack/repos/builtin/packages/fj/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import re
+
+import llnl.util.tty as tty
+import spack.compiler
+import spack.util.executable
+
+
+class Fj(Package):
+    """The Fujitsu compiler system is a high performance, production quality
+    code generation tool designed for high performance parallel
+    computing workloads.
+    """
+
+    homepage = "https://www.fujitsu.com/global/Images/openmp-compiler-datasheet-v5.pdf"
+
+    maintainers = ['t-karatsu']
+
+    def install(self, spec, prefix):
+        raise InstallError(
+            'Fujitsu compilers are not installable yet, but can be '
+            'detected on a system where they are supplied by vendor'
+        )
+
+    executables = ['fcc', 'FCC', 'frt']
```

Comment:
```suggestion
    executables = ['^fcc', '^FCC', '^frt']
```
Fujitsu MPI has mpifcc, mpiFCC and mpifrt as compiler wrapper.